utils/spdx/build.gradle.kts

```diff
@@ -286,30 +286,16 @@ fun generateEnumClass(
 }
 
 val fixupLicenseTextResources by tasks.registering {
-    val licenseUrlMap: Map<File, URL>? by project.extra
-    val licenseExceptionUrlMap: Map<File, URL>? by project.extra
-
     doLast {
-        (licenseUrlMap.orEmpty() + licenseExceptionUrlMap.orEmpty()).forEach { (targetFile, url) ->
-            val filename = url.path.substringAfterLast('/')
-
-            val downloadedFile = targetFile.resolveSibling(filename)
-
-            // Rename files that are named differently on the server.
-            if (downloadedFile != targetFile) {
-                if (targetFile.exists() && !targetFile.delete()) {
-                    throw GradleException("Failed to delete existing file '$targetFile'.")
-                }
-
-                if (!downloadedFile.renameTo(targetFile)) {
-                    throw GradleException("Failed to rename file '$downloadedFile' to '$targetFile'.")
-                }
+        val resourcePaths = listOf("licenses", "exceptions").map { file("src/main/resources/$it") }
```

Comment:
Commit message nit: You don't mention which plugin you mean.

Reply:
Sorry, now you have a typo in the plugin name.

Reply:
Fixed, I was doing to many things at a time it seems.